modules/multiwallet.js

```diff
@@ -19,34 +20,35 @@ let wallets = [
 ];
 
 function add(wallet) {
-  wallets.push({ name: wallet })
+  wallets.push({ name: wallet });
 }
 
-
 function init() {
   const particlDataDir = cookie.getParticlDatadirPath();
 
   // watch the folders
-  const glob = join(particlDataDir, 'wallet_*');
+  const glob = join(particlDataDir, 'wallets', 'wallet_*');
```

Comment:
The significant change in this file. The wallets are found in a dir 'wallets' inside of the particl data dir, so adding this to the path is necessary in order to find the wallets now.

Reply:
Checking up on this, new behavior.